src/main/java/uk/gov/hmcts/sptribs/common/service/CoreCaseApiService.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.sptribs.common.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.sptribs.config.CaseFlagsConfiguration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.singletonMap;
+
+@Service
+public class CoreCaseApiService {
+
```

Comment:
Maybe change the name of this to CcdSupplementaryDataService or somesuch; CoreCaseApiService is something else and this is likely to confuse.